apple-pie/styles/style.css

```diff
@@ -0,0 +1,42 @@
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    max-width: 800px;
+    margin: 0 auto;
+}
+#apple-pie {
+    background-image: url(/apple-pie/images/apple-pie.jpg);
```

Comment:
` background-image: url('../images/apple-pie.jpg'); ` that way you would get the image